http-add-on/templates/interceptor/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.interceptor.pdb.enabled -}}
+{{- if .Capabilities.APIVersions.Has "policy/v1/PodDisruptionBudget" }}
+apiVersion: policy/v1
+{{- else }}
+apiVersion: policy/v1beta1
+{{- end }}
```

Comment:
> policy/v1beta1 PodSecurityPolicy is deprecated in v1.21+, unavailable in v1.25.

I think it's fair to say we should not support it even

Reply:
Great! I'll remove